### PR TITLE
APM > .NET > Add configuration precedence image

### PR DESCRIPTION
**Pull request by jaycdave88**
### What does this PR do?
Adds a new graphic to aid customers with settings the configuration for .NET

### Preview

https://docs-staging.datadoghq.com/jay.dave/aspnet_configuration_image_addition/tracing/setup_overview/setup/dotnet-framework/?tab=environmentvariables#configuration

https://docs-staging.datadoghq.com/jay.dave/aspnet_configuration_image_addition/tracing/setup_overview/setup/dotnet-core/?tab=windows#configure-the-tracer

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/setup_overview/setup/dotnet-framework.md [hunk 1]
```diff
@@ -147,10 +147,7 @@ For additional details on custom instrumentation and custom tagging, see [.NET C
 
 The .NET Tracer has configuration settings which you can set by any of these methods:
 
-- Environment variables
-- In the .NET application code
-- In the application's `app.config` or `web.config` file (.NET Framework only)
-- Using a `datadog.json` file
+![image](https://user-images.githubusercontent.com/7599081/119893905-cca92e00-bef0-11eb-9642-f3402a9e1809.png)
```
Comment (lucaspimentel) [inline content/en/tracing/setup_overview/setup/dotnet-framework.md:150]: Should this image be added to the repository as part of this PR? Probably in `/static/images/tracing/setup/dotnet`?

Comment (lucaspimentel) [reply content/en/tracing/setup_overview/setup/dotnet-framework.md:150]: You'll want to use this syntax (copied from line 103 above):
```
{{< img src="tracing/setup/dotnet/<FILENAME>" alt="<SHORT DESCRIPTION>"  >}}
```

Comment (jaycdave88) [reply content/en/tracing/setup_overview/setup/dotnet-framework.md]: All set - please take another look

**Review by lucaspimentel — APPROVED** [review]

**Review by ruthnaebeck — APPROVED** [review]
👍

File: content/en/tracing/setup_overview/setup/dotnet-framework.md (hunk 1 above)
Comment (lucaspimentel) [reply content/en/tracing/setup_overview/setup/dotnet-framework.md]: Thanks!

**Pull request merged**